gcloud/bigtable/test_column_family.py

```diff
@@ -395,6 +395,48 @@ def test___ne__(self):
         column_family2 = self._makeOne('column_family_id2', None)
         self.assertNotEqual(column_family1, column_family2)
 
+    def test_delete(self):
+        from gcloud.bigtable._generated import (
+            bigtable_table_service_messages_pb2 as messages_pb2)
+        from gcloud.bigtable._generated import empty_pb2
+        from gcloud.bigtable._testing import _FakeStub
+
+        project_id = 'project-id'
+        zone = 'zone'
+        cluster_id = 'cluster-id'
+        table_id = 'table-id'
+        column_family_id = 'column-family-id'
+        timeout_seconds = 7
+        table_name = ('projects/' + project_id + '/zones/' + zone +
+                      '/clusters/' + cluster_id + '/tables/' + table_id)
+        column_family_name = table_name + '/columnFamilies/' + column_family_id
+
+        client = _Client(timeout_seconds=timeout_seconds)
+        table = _Table(table_name, client=client)
+        column_family = self._makeOne(column_family_id, table)
+
+        # Create request_pb
+        request_pb = messages_pb2.DeleteColumnFamilyRequest(
+            name=column_family_name)
+
+        # Create response_pb
+        response_pb = empty_pb2.Empty()
```

Comment:
As with #1322, I'm not sure where our code actually checks / uses the response protobuf.